app/services/scene-collections/scene-collections-api.ts

```diff
@@ -0,0 +1,34 @@
+export interface ISceneCollectionsServiceApi {
+  load(id: string, shouldAttemptRecovery?: boolean): Promise<void>;
+  create(name?: string, setupFunction?: () => void): Promise<void>;
```

Comment:
We can't expose functions as arguments to the API. You can just omit the second param here.

Reply:
👍 It should be used over the API anyway.  It's just a hack for the OBS importer to use.